core/sql/executor/ex_sort.cpp

```diff
@@ -364,12 +365,15 @@ ExSortTcb::ExSortTcb(const ExSortTdb & sort_tdb,
         sortType_.useIterQSForRunGeneration_ = 1;
         break;
   }   
+
+  sortHeap_ = new(getHeap()) NAHeap("Sort Heap", (NAHeap *)getHeap(), 204800);
```

Comment:
Allocation of sortHeap_ here seems to be duplicate allocation since sortHeap_ is already allocated at line 317.  This line can be removed.

Reply:
Done